plugins/scaffolder/src/next/Router/Router.tsx

```diff
@@ -46,6 +46,7 @@ export type NextRouterProps = {
   };
   groups?: TemplateGroupFilter[];
   transformErrors?: ErrorTransformer;
+  noHtml5Validate?: boolean;
```

Comment:
We have documentation explaining how to customize [actions](https://backstage.io/docs/features/software-templates/writing-custom-actions), [fields](https://backstage.io/docs/features/software-templates/writing-custom-field-extensions) and [step](https://backstage.io/docs/features/software-templates/writing-custom-step-layouts) layouts. How about adding one for the form customization as well?